src/u2f.c

```diff
@@ -269,16 +276,39 @@ static void _register(const USB_APDU* apdu, Packet* out_packet)
         return;
     }
 
+    if (!memory_is_initialized() || keystore_is_locked()) {
```

Comment:
in case of `!memory_is_initialized()`, a status screen with the message that the device is not setup would be helpful to the user.

Reply:
fixed, that took longer than I want to admit ;)